lldb/source/DataFormatters/TypeSynthetic.cpp

```diff
@@ -218,10 +218,11 @@ bool ScriptedSyntheticChildren::FrontEnd::MightHaveChildren() {
   return m_interpreter->MightHaveChildrenSynthProviderInstance(m_wrapper_sp);
 }
 
-size_t ScriptedSyntheticChildren::FrontEnd::GetIndexOfChildWithName(
-    ConstString name) {
+llvm::Expected<size_t>
+ScriptedSyntheticChildren::FrontEnd::GetIndexOfChildWithName(ConstString name) {
   if (!m_wrapper_sp || m_interpreter == nullptr)
-    return UINT32_MAX;
+    return llvm::createStringError("Cannot find index of child '%s'",
+                                   name.AsCString());
```

Comment:
Should we be more descriptive with where the error came from? E.g., "ScriptedSyntheticChildren::FrontEnd cannot find index of child '%s'"? Same with the other places in this PR

Reply:
I ended up editing the message everywhere it was used, adding `'classname'` at the beginning of each.